CHANGELOG.md

```diff
@@ -152,6 +152,9 @@
 - fixed unchanged text of button when user requests subscription [#985](https://github.com/eclipse-tractusx/portal-frontend/pull/985)
 - fixed height for "Admin Service Detail" page content [#1001](https://github.com/eclipse-tractusx/portal-frontend/pull/1001)
 
+- **Display Uplaoded file error text in MB**:
+  - Fixed error text of uplaoded file from Byte to MB [#866](https://github.com/eclipse-tractusx/portal-frontend/pull/1185)
```

Comment:
could you please move the update of the changelog to the unreleased section? (first rebase to main)

```suggestion
- **File Upload**:
  - changed error text of uploaded file from Byte to MB [#866](https://github.com/eclipse-tractusx/portal-frontend/pull/1185)
```

Reply:
updated.